buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-Any-Platform

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+ARGS = ['SDK_VERSION', 'PLATFORM', 'TESTS_TARGETS']
```

Comment:
The current pipeline jobs don't use an argument for TEST_TARGETS, they get the default from the variables-function file
https://github.com/eclipse/openj9/blob/master/buildenv/jenkins/common/variables-functions#L267

Is this an intended change in beaviour?

Reply:
on second thought we can dispense of it